Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -1767,6 +1767,42 @@ class ComputeApproximateArea_3 {
 
 }; /* end Kernel::ComputeApproximateArea_3 */
 
+  
+/*!
+  \ingroup PkgKernel23ConceptsFunctionObjects
+  \cgalConcept
+
+  \cgalRefines `AdaptableFunctor`
+
+*/
+class ComputeApproximateAngle_3 {
+public:
+
+  /// \name Operations
+  /// A model of this concept must provide:
+  /// @{
+
+  /*!
+    returns an approximation of the angle between `u` and `v`.
+    The angle is given in rad.
+    \pre `u`and `v` are not equal to the null vector.
+  */
+  Kernel::FT operator()(const Kernel::Vector_3& u,
+                        const Kernel::Vector_3& v) const;
+  
+  /*!
+    returns an approximation of the angle between `p-q` and `r-q`.
+    The angle is given in rad.
```

Comment:
radians

Reply:
in `[0,Pi]`?

Reply:
I fixed degree radian, but didn't add the interval as we do not write that at other places.

Reply:
I think it is important to have the range of possible values.

Reply:
+ rad. -> radians

Reply:
I fixed the rad to radians.  I didn't fix the range.  Why is there no range for the dihedral function between two triangles sharing an edge then?  Shall I add it there ?